scripts/build_workflow_release.sh

```diff
@@ -41,15 +40,17 @@ function deploy_options() {
   # Some workflows have per-environment options, and others share options across all environments.
   local base_options
   if [ -f ${wdl_dir}/${prefix}.options.json ]; then
-    base_options=${prefix}.options.json
+    base_options=${wdl_dir}/${prefix}.options.json
   elif [ -f ${wdl_dir}/${prefix}.${env}.options.json ]; then
-    base_options=${prefix}.${env}.options.json
+    base_options=${wdl_dir}/${prefix}.${env}.options.json
+  elif [[ "${wdl_dir}" =~ .*"skylab".* ]]; then
+    base_options=${DSDE_PIPELINES_ROOT}/tests/skylab/test.options.json
```

Comment:
```suggestion
    base_options=${DSDE_PIPELINES_ROOT}/tests/skylab/options.json
```

Reply:
I thought I changed the name of the file in that folder to test.options.json though?

Reply:
you did, you're right! Ignore me